packages/webpack-cli/README.md

```diff
@@ -68,9 +68,9 @@ npx webpack-cli --help verbose
   help|h [command] [option]                  Display help for commands and options.
   info|i [options]                           Outputs information about your system.
   init|c [generation-path] [options]         Initialize a new webpack project.
-  loader|l [output-path]                     Scaffold a loader.
+  loader|l [output-path] [options]           Scaffold a loader.
   migrate|m <config-path> [new-config-path]  Migrate a configuration to a new version.
-  plugin|p [output-path]                     Scaffold a plugin.
+  plugin|p [output-path] [options]           Scaffold a plugin.
```

Comment:
Was it fixed manually? Because if we have problems with indent, we should fix it

Reply:
> Was it fixed manually?

Yup

Reply:
We need fix it, I will look at this

Reply:
I added the entry manually at first which broke the indentation. It got fixed once I grabbed the help output. I think we are good here.

Reply:
Yep you are right